src/DataFixtures/ORM/LoadIdeaThreadCommentData.php

```diff
@@ -91,6 +93,13 @@ public function load(ObjectManager $manager)
             new \DateTime('-1 minute')
         );
 
+        $commentOfAdherent6OnIdeaWaste = ThreadComment::create(
+            Uuid::fromString(self::THREAD_COMMENT_09_UUID),
+            'Commentaire d`un adhérent',
```

Comment:
Can you use `\'` or `"Commentaire d'un adhérent",` instead of the ` char please ?